web/client/components/geostory/common/enhancers/withIdentifyPopup.jsx

```diff
@@ -59,7 +59,7 @@ export const withIdentifyRequest  = mapPropsStream(props$ => {
                     const appParams = filterRequestParams(layer, includeOptions, excludeParams);
                     const param = { ...appParams, ...requestParams };
                     const reqId = uuidv1();
-                    return getFeatureInfo(basePath, param, false)
+                    return getFeatureInfo(basePath, param, false, undefined, layer)
```

Comment:
Maybe you can reverse the last two parameters in getFeatureInfo so you can avoid remove here.